### DISCUSSION TITLE
Can't find osgEarth during configuration

### DISCUSSION THREAD

**Discussion by phdavis1027** [omnetpp/omnetpp#1104] [Q&A] [answered]
I am trying to install Omnetpp with Qtenv from source on Ubuntu 22.04. When running `./configure,` I get the error:

```
Cannot find osgEarth 2.7 or later...
```

I have installed osgEarth from source and confirmed manually that the relevant headers libraries are in `/usr/lib/osgearth`. Additionally, I have tried editing `OSGEARTH_LIBS` in `configure.user` as suggested [here](https://groups.google.com/g/omnetpp/c/OVx_5xDY5fM). With and without these modifications, `config.log` indicates that clang++ couldn't find the libraries. What should my next steps be?

**Comment by rhornig** [comment]
could you attach the config.log?

**Comment by phdavis1027** [comment]
[config.log](https://github.com/omnetpp/omnetpp/files/14362056/config.log)
Voila the file.

**Reply by rhornig** [reply, marked as answer]
Are you absolutely sure that the `libosgEarth` and `libosgEarthUtils` libraries are properly installed in their default location (in `/usr/lib/x86_64-linux-gnu`). Because the config script is not able to link with those libs according to the logs. Also the osgEarth plugin must be properly installed in `/usr/lib/x86_64-linux-gnu/osgPlugins-3.6.5` among the other open-scene-graph plugins.

Also note that only osgEarth 2.x is supported and the latest version seems to be 3.5 which is a lot newer...

**Reply by phdavis1027** [reply]
They indeed were not in that location, just in `/usr/lib`. Shouldn't clang be able to find them there? In any case, when I symlinked the contents of `/usr/lib/osgearth` to the plugin directory you mentioned, config.log did indeed find the libosgEarth. However, it appears that libosgEarthUtils wasn't built, which may be an issue with the versioning you mentioned. I'll build version 2.7 and try again.

**Reply by rhornig** [reply]
2.9 or 2.10 should do just fine. (but not any 3.x version).

**Reply by phdavis1027** [reply]
This worked to get an installation, and when I ran the aloha sample the IDE started up just fine, but when I just try to run `omnetpp` from Bash I'm told the IDE is not installed.

**Reply by rhornig** [reply]
Are you sure you installed the **linux** specific archive and not the **core** one? Check whether you have the `ide` folder in the omnetpp root directory.

**Reply by phdavis1027** [reply]
Indeed it was not. I re-installed and now have everything I want. Thanks for the hand-holding!

**Reply by phdavis1027** [reply]
This is a separate issue, but maybe you could provide some clarity on how to increase the JVM size when running the IDE. I apparently don't even have Eclipse or Java installed on my research machine so I'm rather confused about where the binaries that Omnetpp is using even are.

**Reply by rhornig** [reply]
The IDE (eclipse based) is in the 'ide' folder and the `ide/opp_ide.ini` file contains the JVM startup parameters. It is already configured with a 1Gb limit which should be enough for most cases, but you can increase that there.
